components/layout_2020/positioned.rs

```diff
@@ -396,91 +476,100 @@ impl<'box_tree> HoistedAbsolutelyPositionedBox<'box_tree> {
             block_end: block_axis.margin_end,
         };
 
-        let for_icb = for_initial_containing_block;
-        PositioningContext::for_positioned(layout_context, for_icb, |positioning_context| {
-            let size;
-            let fragments;
-            match self.absolutely_positioned_box.contents.as_replaced() {
-                Ok(replaced) => {
-                    // https://drafts.csswg.org/css2/visudet.html#abs-replaced-width
-                    // https://drafts.csswg.org/css2/visudet.html#abs-replaced-height
-                    let style = &self.absolutely_positioned_box.contents.style;
-                    size = replaced_used_size.unwrap();
-                    fragments = replaced.make_fragments(style, size.clone());
-                },
-                Err(non_replaced) => {
-                    // https://drafts.csswg.org/css2/visudet.html#abs-non-replaced-width
-                    // https://drafts.csswg.org/css2/visudet.html#abs-non-replaced-height
-                    let inline_size = inline_axis.size.auto_is(|| {
-                        let available_size = match inline_axis.anchor {
-                            Anchor::Start(start) => {
-                                cbis - start - pb.inline_sum() - margin.inline_sum()
-                            },
-                            Anchor::End(end) => cbis - end - pb.inline_sum() - margin.inline_sum(),
+        let for_cbfad = for_nearest_cbfad;
```

Comment:
This isn't changed other than the replacement of `PositioningContext::for_positioned` with `PositioningContext::create_and_layout_positioned`, right?

Reply:
That's correct. Unfortunately this caused a reflow of the entire block.